project/CelebornBuild.scala

```diff
@@ -202,7 +213,7 @@ object CelebornCommon {
         "io.dropwizard.metrics" % "metrics-core" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-graphite" % metricsVersion,
         "io.dropwizard.metrics" % "metrics-jvm" % metricsVersion,
-        "io.netty" % "netty-all" % nettyVersion,
+        ("io.netty" % "netty-all" % nettyVersion).exclude("io.netty", "netty-handler-ssl-ocsp"),
```

Comment:
can you remove this change? as we discussed before, do exclusion after introducing the dependency list audit mechanism.

Reply:
updated